### PR TITLE
(feat) HSC-286: Add visit attributes 'Mode of arrival' and 'Level of emergency severity assessment'

### PR DESCRIPTION
**Pull request by vasharma05**
This PR adds 2 new visit attributes
1. Mode of arrival
2. Level of emergency severity assessment

### CODE REVIEW

File: configs/openmrs/initializer_config/attributetypes/attribute_types.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Entity name,Name,Description,Min occurs,Max occurs,Datatype classname,Datatype config,Preferred handler classname,Handler config,_order:1000
+a56959b5-4fe2-4d69-a553-fdc77957c48b,,Visit,Mode of arrival,The mode of arrival that the patient took to the location,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,f61a8a18-cb64-4a63-bc28-56b4789941ff,,,
+d63554a8-4979-4719-a918-1ccc3d924ed7,,Visit,Level of emergency severity assessment,Level of emergency severity assessment,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,bbb02557-0277-4979-a1fa-b0020f136066,,,
```
Comment (vasharma05) [inline configs/openmrs/initializer_config/attributetypes/attribute_types.csv:3]: Hi @kazlaw!
The `Level of emergency severity assessment` concept is being used in the `triage.json` form, and I have used the same UUID for the above visit attribute type's `Datatype config` mapping. But I didn't find the concept defined anywhere, do we need to define the concept mentioned above?
Thanks!

File: configs/openmrs/initializer_config/concepts/visit_attributes.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:500
+2fdfe136-263c-4d59-b96e-ae4a76d9445f,,,Minibus,Minibus,Minibus,Minibus,,,Misc,N/A,,,,,,,,,,,
+93298e19-1e9a-4306-beec-c134d0989a0a,,,Ambulated,Ambulated,Ambulated,Ambulated,,,Misc,N/A,,,,,,,,,,,
+693c7001-1433-4ebb-aee9-1ab0dd1787f6,,,Car,Car,Car,Car,,,Misc,N/A,,,,,,,,,,,
+f61a8a18-cb64-4a63-bc28-56b4789941ff,,,Mode of arrival,Mode of arrival,Mode of arrival,Mode of arrival,,,Question,Coded,,Minibus;Ambulated;Car;Other,,,,,,,,,
```
Comment (vasharma05) [inline configs/openmrs/initializer_config/concepts/visit_attributes.csv:5]: @kazlaw, I have created new UUIDs for these concepts, should we use the same UUIDs which were being used in the older implementation?
Thanks!

**Review by vasharma05 — COMMENTED** [review]
@kazlaw, I have a few questions, please have a look. Alongside, the french translations are missing for all the concepts' texts.
Thanks!

**Review by kazlaw — APPROVED** [review]
image


Looks good

**Pull request merged**